asistente_ladm_col/utils/qgis_utils.py

```diff
@@ -631,12 +631,19 @@ def reset_automatic_field(self, db, layer, field):
     def set_automatic_fields(self, db, layer, layer_name):
         self.set_automatic_fields_namespace_local_id(db, layer_name, layer)
 
+        list_dicts_field_expression = list()
+
         if layer.fields().indexFromName(db.names.VERSIONED_OBJECT_T_BEGIN_LIFESPAN_VERSION_F) != -1:
-            self.configure_automatic_fields(db, layer, [{db.names.VERSIONED_OBJECT_T_BEGIN_LIFESPAN_VERSION_F: "now()"}])
+            list_dicts_field_expression.append({db.names.VERSIONED_OBJECT_T_BEGIN_LIFESPAN_VERSION_F: "now()"})
+
+        if layer.fields().indexFromName(db.names.T_ILI_TID_F) != -1:
+            list_dicts_field_expression.append({db.names.T_ILI_TID_F: "uuid()"})
```

Comment:
It's now obsolete. It's better to rebase on master than reproducing a single commit.